Problem2.py

```diff
@@ -0,0 +1,43 @@
+"""
+88. Merge Sorted Array
+Time Complexity - O(n+m)
+Space Complexity - O(1)
+Check if given list is not empty
+Here we are checking condition from Last element and then going toward small index i = m-1, j = n-1
+While loop verifying if index of both the list doesnt go out of bound
+    if value for nums2 is reater than value of nums1
+        copy nums2 value in k = m+n-1 location
+            decrement index of j and k
+    else
+    copy nums1 value in k = m+n-1 location
+            decrement index of i and k
+if nums2 j is greater than or equal to 0 that means values of nums1 has been completely copied
+Using while loop copy rest of elements
+    """
+
+class Solution:
+    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
```

Comment:
Good Job! One request would be to add if the code was accepted on LeetCode. Thanks